src/review/modules/components/ProductFooter.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles, createStyleSheet } from 'material-ui/styles'
+import Typography from 'material-ui/Typography'
+import Divider from 'material-ui/Divider'
+import { fullWhite, darkWhite, lightWhite } from 'material-ui/styles/colors'
+import LayoutBody from 'modules/components/LayoutBody'
+import Link from 'modules/components/Link'
+import GitHub from 'review/modules/components/GitHub'
+
+const styleSheet = createStyleSheet('ProductFooter', theme => ({
+  root: {
+    background: theme.brandColor,
+    color: darkWhite,
+    overflow: 'auto',
+  },
+  title: {
+    color: fullWhite,
+  },
+  divider: {
+    margin: `${theme.spacing.unit * 2}px 0`,
+    background: 'rgba(255, 255, 255, 0.2)',
+  },
+  copyright: {
+    color: lightWhite,
+  },
+  list: {
+    margin: 0,
+    paddingLeft: 0,
+    listStyle: 'none',
+  },
+  icon: {
+    width: 16,
+    height: 16,
+    marginRight: theme.spacing.unit,
+  },
+}))
+
+function ProductFooter(props) {
+  const {
```

Comment:
I like that pattern.
- Already used everywhere, if we change it, we need to do it everywhere.
- Easier to refactor into a class
- Now, I'm used to it with Material-UI